it/uk/gov/hmrc/incorporatedentityidentificationfrontend/controllers/CheckYourAnswersControllerISpec.scala

```diff
@@ -48,12 +61,14 @@ class CheckYourAnswersControllerISpec extends ComponentSpecHelper
         businessEntity = LimitedCompany
       ))
       stubAuth(OK, successfulAuthResponse(Some(testInternalId)))
+      stubAudit()
       stubRetrieveCompanyProfileFromBE(testJourneyId)(status = OK, body = Json.toJsObject(testCompanyProfile))
       stubRetrieveCtutr(testJourneyId)(status = OK, body = testCtutr)
 
       lazy val result: WSResponse = get(s"$baseUrl/$testJourneyId/check-your-answers-business")
 
       result.status mustBe OK
+      verifyAudit()
```

Comment:
Could this verify actually verify the correct information is audited please?

Reply:
I think this is done at unit test level

Reply:
Yes, can do.  I lifted this from Soletrader.  There are some unit tests that test what the audit service produces.

Reply:
I've fixed this with a new method that verifies the explicit audit sent